apps/core/templates/core/home-authenticated.html

```diff
@@ -22,6 +22,30 @@ <h2>{% trans "Welcome to Pixel" %}</h2>
       <div id="pixels-by-omics-unit-type" style="height: 500px"></div>
     </div>
 
+    <div class="cell">
+      <p style="font-family:'Arial'; font-size: 15px;font-weight: bold; color:black;text-align:center;">
```

Comment:
Can you get rid of this inline `style`? Either dont style it right now or put this in a SCSS file.